artiq/gateware/eem_7series.py

```diff
@@ -19,6 +19,21 @@ def peripheral_dio(module, peripheral, **kwargs):
         edge_counter_cls=edge_counter_cls, **kwargs)
 
 
+def peripheral_dio_spi(module, peripheral, **kwargs):
+    ttl_classes = {
+        "input": ttl_serdes_7series.InOut_8X,
+        "output": ttl_serdes_7series.Output_8X
+    }
+    if len(peripheral["ports"]) != 1:
+        raise ValueError("wrong number of ports")
+    spi = [(s["clk"], s.get("mosi"), s.get("miso"), s.get("cs", []))
+           for s in peripheral["spi"]]
+    ttl = [(t["pin"], ttl_classes[t["direction"]],
+            edge_counter.SimpleEdgeCounter if t.get("edge_counter") else None)
```

Comment:
```suggestion
            edge_counter.SimpleEdgeCounter if t.get("edge_counter", False) else None)
```

Reply:
Your proposed change does not change the functionality as `dict.get()` already returns `None`.